ProfileView/src/index.ts

```diff
@@ -384,54 +404,54 @@ export default class CjaasProfileWidget extends LitElement {
     const tabs = this.profile.filter((x: any) => x.query.type === "tab");
     // TODO: Track the selected tab to apply a class to the badge for color synching, making blue when selected
     debugger;
-    const activityTab =
-      this.profileReadToken || this.profileWriteToken
-        ? html`
-            <md-tab slot="tab">
-              <span>All</span>
-            </md-tab>
-            <md-tab-panel slot="panel">
-              ${this.renderTimeline(this.timelineItems)}
-            </md-tab-panel>
-          `
-        : html`
-            <div class="center full-height">
-              <slot name="l10n-no-data-message">
-                <div>No data to show</div>
-              </slot>
-            </div>
-          `;
+    const activityTab = this.profileWriteToken
+      ? html`
+          <md-tab slot="tab">
+            <span>All</span>
+          </md-tab>
+          <md-tab-panel slot="panel">
+            ${this.renderTimeline(this.timelineItems)}
+          </md-tab-panel>
+        `
+      : html`
+          <div class="center full-height">
+            <slot name="l10n-no-data-message">
+              <div>No data to show</div>
+            </slot>
+          </div>
+        `;
     if (tabs && tabs.length > 0) {
       return html`
         <md-tabs>
-          ${activityTab}
-          ${tabs.map((x: any) => {
-            return html`
-              <md-tab slot="tab">
-                <span>${x.query.DisplayName}</span
-                ><md-badge small
-                  >${x.journeyEvents ? x.journeyEvents.length : "0"}</md-badge
-                >
-              </md-tab>
-              <md-tab-panel slot="panel">
-                <!-- use verbose journey events with timeline comp -->
-                ${x.journeyEvents
-                  ? this.renderTimeline(
-                      x.journeyEvents.map((y: any) =>
-                        this.getTimelineItemFromMessage(y)
-                      )
-                    )
-                  : nothing}
-              </md-tab-panel>
-            `;
-          })}
+          ${activityTab} ${tabs.map((x: any) => {})}
         </md-tabs>
       `;
     } else {
       return activityTab;
     }
   }
 
+  getTab(tab: any) {
```

Comment:
I see this replaces . . . however it is not called anywhere. Safe to say this is work in progress?

Reply:
Let me fix this.

Reply:
This is fixed